LiskHQ/lisk-hub
# Add account initialization - Closes #284

### What was the problem?
We needed an account initialisation

### How did I fix it?
we had none

### How to test it?
Log into an account with only incoming transactions -> you should see the account init option in the wallet and dashboard, then init your account -> the notice should be gone after

### What it looks like
![screen shot 2018-03-15 at 10 47 12](https://user-images.githubusercontent.com/9592216/37456278-450c7f6a-283f-11e8-9da2-c10d61e82e05.png)

<img width="1675" alt="screen shot 2018-03-13 at 10 47 54" src="https://user-images.githubusercontent.com/9592216/37337042-773b9b2a-26b3-11e8-9e2b-f29b4b8d9aab.png">
<img width="1674" alt="screen shot 2018-03-13 at 10 48 21" src="https://user-images.githubusercontent.com/9592216/37337043-776a123e-26b3-11e8-85ca-fe4d8ae07ba3.png">



### Review checklist
- The PR solves #284
- All new code is covered with unit tests
- All new features are covered with e2e tests
- All new code follows best practices
